Database/Corrections/Classic/classicItemFixes.lua

```diff
@@ -25,7 +25,6 @@ function QuestieItemFixes:Load()
         [1262] = {
             [itemKeys.relatedQuests] = {116,117},
             [itemKeys.npcDrops] = {239},
-            [itemKeys.objectDrops] = {},
```

Comment:
Are you sure this clearance of the `objectDrops` and the other ones you removed are not required? If the database has entries then they will show up again.

Reply:
I looked in the file https://github.com/Questie/Questie/blob/master/Database/Classic/classicItemDB.lua and that field was already `nil`, unless there's another place I should check?